app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -66,6 +72,22 @@ class ResetProdTask extends ApiRequestHandler<Body> {
           .map((LuciBuilder builder) => builder.name)
           .single;
     }
+    final RepositorySlug slug = RepositorySlug(owner, repo);
+    final Iterable<Build> currentBuilds = await luciBuildService.getProdBuilds(slug, commit.sha, builder, repo);
+    final List<Status> noReschedule = <Status>[Status.started, Status.scheduled, Status.success];
+    final Build build = currentBuilds.firstWhere(
```

Comment:
Do we need to check every build in `currentBuilds`, or checking the first is enough (similar to try case)?

Reply:
The first one is enough. Ideally if there is a started, succeeded and or queued task we should not trigger a new one.